### DISCUSSION TITLE
Packet radio soft-modem for low-speed data transfers between 2 x6100 and ethernet / wifi hotspot

### DISCUSSION THREAD

**Discussion by jasonmtravis** [gdyuldin/x6100_gui#18] [Ideas]
This is the reason that I asked about getting the WIFI added. 

We've all seen disasters that take out communication infrastructure. Amateur operators always jump in to help relay via voice traffic and CW, but the whole traditional packet setup can be bulky to transport. 
As the FT8/FT4 implementation proves, this handheld rig has the guts to do packet. It may be QRP, but an external PA can give it the boosted range to be more effective. As long as the traffic isn't encrypted the FCC should have issues.

Operators could use USB-C to ethernet adapters to connect wired equipment, or if a WIFI hotspot feature were available via wireless connection.

https://hackaday.io/project/164092/files
https://www.complete.org/packet-radio-on-hf/
https://www.complete.org/linux-packet-radio/

73,
Jason - W5JMT

**Comment by gdyuldin** [comment]
Well, I guess, most disadvantage of using packet radio protocol during disasters is the it's not very common. But let this idea be here.

**Comment by jasonmtravis** [comment]
My local ARES groups are experimenting with using SSTV for photos of damage assessments. SSTV tends to be pretty lossy even at close range. With packet radio, or better yet, the "New Packet Radio" the protocol allows retransmits and faster throughput while still staying within the FCC max bandwidth rulings for 70cm. It's probably a lot of work, and a small use case.
